story.py

```diff
@@ -13,5 +13,6 @@ def read_tsv():
 
 def main():
 	read_tsv()
+	user_input = input()
```

Comment:
You should still put this into its own function, then call that in `main()`. It's only one line now, but you know you will have to manipulate this variable later to manage the story flow, so there's no reason not to separate it out now.